setup.py

```diff
@@ -0,0 +1,45 @@
+from setuptools import find_packages
+from setuptools import setup
+
+package_name = 'ros2launch_security'
+
+setup(
+    name=package_name,
+    version='0.1.0',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/' + package_name, ['package.xml']),
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+    ],
+    install_requires=['ros2launch', 'launch_ros', 'setuptools'],
+    zip_safe=True,
+    author='Joe Example',
+    author_email='joe@example.com',
+    maintainer='Joe Example',
+    maintainer_email='joe@example.com',
```

Comment:
@gbiggs not sure if Joe should be a maintainer.

Reply:
Good catch! @kyrofa was going to be maintainer of this package, but since he's left Canonical I expect he's no longer available. I don't think there is anyone else lined up yet. I will need to ask the Security WG who can be maintainer before this can be merged.